ci-operator/config/openshift/cluster-api-actuator-pkg/openshift-cluster-api-actuator-pkg-master.yaml

```diff
@@ -104,6 +134,22 @@ tests:
         requests:
           cpu: 100m
     workflow: ipi-gcp
+- as: e2e-gcp-capi-techpreview
+  skip_if_only_changed: ^docs/|^testutils/|\.md$|^(?:.*/)?(?:\.gitignore|OWNERS|PROJECT|LICENSE)$
+  steps:
+    cluster_profile: gcp
+    env:
+      FEATURE_SET: TechPreviewNoUpgrade
+    test:
+    - as: test
+      cli: latest
+      commands: JUNIT_DIR=${ARTIFACT_DIR} make -C /go/src/github.com/openshift/cluster-api-actuator-pkg
```

Comment:
I see this is different, any particular reason on why?

Reply:
No particular reason. I don't aim to change anything, so I just copied from current jobs [e2e-aws-operator](https://github.com/openshift/release/blob/master/ci-operator/config/openshift/cluster-api-actuator-pkg/openshift-cluster-api-actuator-pkg-master.yaml#L53-L65)   [e2e-azure-operator](https://github.com/openshift/release/blob/master/ci-operator/config/openshift/cluster-api-actuator-pkg/openshift-cluster-api-actuator-pkg-master.yaml#L79-L91)    [e2e-gcp-operator](https://github.com/openshift/release/blob/master/ci-operator/config/openshift/cluster-api-actuator-pkg/openshift-cluster-api-actuator-pkg-master.yaml#L92-L106) and added `FEATURE_SET: TechPreviewNoUpgrade`